ext/lib/mgmt/mcumgr/cmd/log_mgmt/Kconfig

```diff
@@ -18,7 +18,6 @@
 menuconfig MCUMGR_CMD_LOG_MGMT
     bool
     prompt "Enable mcumgr handlers for log management"
-    depends on MDLOG
```

Comment:
As a note here, [MCUMGR_CMD_LOG_MGMT](http://docs.zephyrproject.org/reference/kconfig/CONFIG_MCUMGR_CMD_LOG_MGMT.html) (*Enables mcumgr handlers for log management*) will have been impossible to enable until this gets in, from the moment it got merged.